### PR TITLE
Let users dismiss the matrix message

### LINKED ISSUE #489: opt out from "Consider rmoving" messages?
I use Flammie[m]  for matrix account and Flammie for irssi. I don't need new message every day that I can consider removing the [m].
consider-removing

### PR DESCRIPTION
**Pull request by scoopgracie**
This commit allows users to run `.dismiss` to avoid receiving the 'please remove the [m]' message. It also makes the message itself a bit friendlier.
Closes #489.

### CODE REVIEW

**Comment by jonorthwash** [comment]
Could you put the `dismiss` function in the same file as the greeter?

File: modules/dismiss.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+import json
+import os
+def dismiss(phenny, input):
+    try:
+        opted_out_list = open(os.getenv('HOME') + '/.phenny/opted_out_of_matrix_message.json', 'r')
```
Comment (jonorthwash) [inline modules/dismiss.py]: This is the wrong way to save data for phenny.  Please check how other modules do it.

Comment (scoopgracie) [reply modules/dismiss.py]: Use SQLite? That seems a bit much for a simple list.

**Comment by scoopgracie** [comment]
Will do later in moving dismiss function.

File: modules/dismiss.py (hunk 1 above)
Comment (scoopgracie) [reply modules/dismiss.py]: Isn't SQL meant for when you have a group of keys and values, not for lists? It seems almost like a hack to use "SELECT FROM opted_out WHERE (nick = 'somenick')" and check if the length was 0.

Comment (scoopgracie) [reply modules/dismiss.py]: OK, will do!

**Comment by scoopgracie** [comment]
Fixed the issues.

**Comment by Androbin** [comment]
Someone fix the tests please

**Comment by scoopgracie** [comment]
Fixing it in a new PR.

**Pull request merged**